openghg/standardise/_standardise.py

```diff
@@ -119,6 +120,9 @@ def standardise_surface(
             See https://zarr.readthedocs.io/en/stable/api/codecs.html for more information on compressors.
         filters: Filters to apply to the data on storage, this defaults to no filtering. See
             https://zarr.readthedocs.io/en/stable/tutorial.html#filters for more information on picking filters.
+        chunks: Chunk schema to use when storing data the NetCDF. It expects a dictionary of dimension name and chunk size,
+            for example {"time": 100}. If None then a chunking schema will be set automatically by OpenGHG as per the <link to documentation>.
```

Comment:
Would it be worth adding explicit TODO lines within all the doc-strings to make sure we remember to add details for "link to documentation" before we finish off all the changes?

Reply:
Done